### DISCUSSION TITLE
Expanding Price graph.

### DISCUSSION THREAD

**Discussion by Laziegoblin** [iotaledger/firefly#1667] [Features / Enhancements]
Hi,

I wondered if it would be possible to increase the timeframe a little.
1 month price timeframe is pretty short. 
It would be great to have a "Year" and "Max" option.

Regards,
Lazie

**Comment by MiKeF05** [comment]
Yes this has already been reported and is on the discussion list (https://github.com/iotaledger/firefly/discussions/1529).

I think it would be good to close this discussion as it is duplicated.
